### PR TITLE
🎨 Create Home Page Overview Section

### PR DESCRIPTION
**Pull request by MarsMwau**
# Description

I have created and styled the overview section to meet the required design.

Fixes #4 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Laptop
Screenshot 2023-09-29 133942

Tablet
Screenshot 2023-10-20 150736

Mini Tablet
Screenshot 2023-10-20 150840

Phone
Screenshot 2023-10-20 150804

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,15 +5,36 @@
...
+    <div class="image-left">
+      <img src="src/assets/image2.png" alt="man">
+    </div>
+    <div class="text-right">
+      <h2>From here to the world</h2>
+      <p>
+        Getting thousands of people up to speed can be hard and expensive. But, it doesn’t have to be.
+        <br />
+
+        <br />
```
Comment (JAPHETHNYARANGA) [inline index.html]: use box model to space

File: index.html [hunk 2]
```diff
@@ -5,15 +5,36 @@
...
+        Elewa is an education partner which develops the kind of training that work at scale. This without losing track
+        of what matters most, people!
+        <br />
+
+        <br />
```
Comment (JAPHETHNYARANGA) [inline index.html]: use box model to space

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
please review you code , theres a horizontal scroll bar , so check on your box model styles , and please attach screenshot of mobile, thanks.

File: index.html (hunk 1 above)
Comment (MarsMwau) [reply index.html]: complete

**Comment by JAPHETHNYARANGA** [comment]
Good attempt but this issue has been closed . Please pick another open issue. Hint; don't delete code already in a file when working on it.

**Pull request closed without merging**